# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: accessible/ipc/other/PDocAccessible.ipdl
# Commit: 5f4d7f8e8c6e
# Full Hash: 5f4d7f8e8c6efdc767fb2d947a1e5b1607a96d28
# Author: James Teh <jteh@mozilla.com>
# Date: 2020-03-18 16:05:52
# Description:
#   Bug 1622751: Teach IPDL that PDocAccessible::AccessibleAtPoint can return a null result doc. r=eeejay
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67081
# ==============================================================================

diff -r 70f8ce3e2d39 -r 5f4d7f8e8c6e accessible/ipc/other/PDocAccessible.ipdl
--- a/accessible/ipc/other/PDocAccessible.ipdl	Wed Mar 18 02:56:31 2020 +0200
+++ b/accessible/ipc/other/PDocAccessible.ipdl	Tue Mar 17 22:13:55 2020 +0000
@@ -305,7 +305,7 @@
   nested(inside_sync) sync URLDocTypeMimeType(uint64_t aID) returns(nsString aURL, nsString aDocType, nsString aMimeType);
 
   nested(inside_sync) sync AccessibleAtPoint(uint64_t aID, int32_t aX, int32_t aY, bool aNeedsScreenCoords, uint32_t aWhich)
-    returns(PDocAccessible aResultDoc, uint64_t aResultID);
+    returns(nullable PDocAccessible aResultDoc, uint64_t aResultID);
 
   nested(inside_sync) sync Extents(uint64_t aID, bool aNeedsScreenCoords)
     returns(int32_t aX, int32_t aY, int32_t aWidth, int32_t aHeight);
